apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { ValueType } from 'react-select'
+
+import { Box, Input, Select, Button } from '@island.is/island-ui/core'
+import {
+  ReactSelectOption,
+  TempCase as Case,
+} from '@island.is/judicial-system-web/src/types'
+import { BlueBox } from '@island.is/judicial-system-web/src/components'
+import { IndictmentCount as TIndictmentCount } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { indictmentCount as strings } from './IndictmentCount.strings'
+
+interface Props {
+  indictmentCount: TIndictmentCount
+  workingCase: Case
+  onChange: (
+    indictmentCountId: string,
+    indictmentCount: TIndictmentCount,
+  ) => void
+  onDelete?: (indictmentCount: TIndictmentCount) => Promise<void>
```

Comment:
Consider passing only the indictment count id to `onDelete`. That should be enough to find the correct count to delete.